content/develop/_index.md

```diff
@@ -10,35 +10,89 @@ tags = [ "theme" ]
+A container never directly styles the children inside the contain. It is only concerned with their horizontal or vertical alignment and spacing. For example in a card, you might apply margins to the slots, but you would not apply padding or color not to the content inside of slots.
 
-**Example: [rh-card](https://github.com/RHElements/rhelements/blob/master/elements/rh-card/src/rh-card.scss)**
+**Example: [Card](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-card/src/pfe-card)**
 
 This component has background color options, and attributes to change the vertical spacing between the slots.
 
-## Content RHElements
+## Content Elements
 
 A component always touches all four sides of its parent container. For example, if you are creating a teaser component, it should not have padding or margins around the type. It might, however have margins between the headline and author.
 
-**Example: [rh-cta](https://github.com/RHElements/rhelements/blob/master/elements/rh-cta/src/rh-cta.scss)**
+**Example: [Call-to-action](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-cta/src/pfe-cta)**
 
-This component has typography styles, and attributes to change the priority level of the call-to-action. It also has `on="dark"` styles built in, so that colors will automatically adjust. 
\ No newline at end of file
+This component has typography styles, and attributes to change the priority level of the call-to-action. It also has `on="dark"` styles built in, so that colors will automatically adjust. 
+
+## Combo Elements
+
+These elements are designed to create easy-to-use combinations of other smaller components. Ideally they also bundle up attributes into a few configurations.
+
+**Example: [Hide-show](https://github.com/patternfly/patternfly-elements/blob/master/elements/pfe-hide-show)**
+
+
+## @todo should features be opt-in or opt-out?
+- If it's both, that’s confusing
+- If they are opt-in, it ensures that components don't change themselves on people's sites who are asking for all minor versions of components.
+
+
+
+
+# Attribute naming 
+
+## General
+- Attribute names should be specific so they are easy to read and understand. 
+- Use standard [HTML global attributes](https://www.w3schools.com/tags/ref_standardattributes.asp) and / or proper [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA) when appropriate
+    - Example: `<pfe-pagination hidden>` `<pfe-cta title="RHEL features">`
+
+## Prefixing attributes
+
+- To avoid collisions with other libraries, known and unknown as well as new HTML tags and attributes
+- To ensure folks who have no prefixing on their existing sites can use PF Elements without worry of collisiding CSS or JavaScript
+- To  have a single-source-of-truth, i.e. one class or attribute for styling vs. chaining attributes
+
+## Optional component features should be activated with a unique attribute instead of values within a generic attribute
```

Comment:
This long title is a little confusing and I think it's been addressed above so I might suggest moving this content into the general section.

Reply:
I moved this up but left the bullets as nested, does this make more sense now?